File: app/src/main/java/com/mgaetan89/showsrage/model/Indexer.java
package com.mgaetan89.showsrage.model;

public enum Indexer {
	TVDB("tvdbid"), TVRAGE("tvrageid");

	private String paramName;

	Indexer(String paramName) {
		this.paramName = paramName;
	}

	public String getParamName() {
		return this.paramName;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/model/Indexer.kt
package com.mgaetan89.showsrage.model

sealed class Indexer(val paramName: String) {
	object TMDB : Indexer("tmdbid")
	object TVDB : Indexer("tvdbid")
	object TVMAZE : Indexer("tvmazeid")
	object TVRAGE : Indexer("tvrageid")
}
